docs/installation/local.md

```diff
@@ -2,169 +2,165 @@
-sudo -u postgres psql
+# Make yourself superuser in PostgeSQL
+sudo -u postgres createuser -s $USER
+# You are already a superuser, you can freely create new database
+createdb oevent -O $USER
+createdb opev_test -O $USER
+```
 
-# For macOS users
+This method is called "ident" method, where the connection is established via "Unix domain socket", which is a file, instead of a pair of IP:port. This method is safe because:
+
+- No password to protect.
+- Only available to same-machine clients, meaning no worries about attack from outside.
+
+### For MAC users
+
+```sh
 psql -d postgres
 ```
 
-* When inside psql, create a user for open-event and then using the user create the database. Also, create a test database named opev_test for the test suites by dumping the oevent database into it. without this, the tests will not run locally.
-
-For ease of development, you should create Postgres user with the same username as your OS account. If your OS login account is _john_, for example, you should create _john_ user in Postgres. By this, you can skip entering password when using database.
+Inside `psql`'s shell:
 
 ```sql
 CREATE USER open_event_user WITH PASSWORD 'opev_pass';
 CREATE DATABASE oevent WITH OWNER open_event_user;
 CREATE DATABASE opev_test WITH OWNER open_event_user;
 ```
 
-* Once the databases are created, exit the psql shell with `\q` followed by ENTER.
+Once the databases are created, exit the psql shell with `\q` followed by ENTER.
 
-
-* **Step 3** - Create application environment variables.
+## Generate configuration
 
 ```sh
 cp .env.example .env
 ```
 
 Add `SECRET_KEY={{something random}}` in .env file for cryptographic usage. Note that server will not run in production mode if you don't supply a secret.
-To get a good secret value, run `python -c 'import secrets;print(secrets.token_hex())'` in a terminal and replace `{{something random}}` with its output in the line above and paste it in `.env` file
-
-
-* **Step 4** - Start the postgres service.
+To get a good secret value, run `python -c 'import secrets;print(secrets.token_hex())'` in a terminal and replace `{{something random}}` with its output in the line above and paste it in `.env` file.
 
-```sh
-sudo service postgresql restart
-```
-for mac users:
+If you created a dedicated PostgreSQL user and password, you should update the *.env* file content.
 
-```sh
-brew services restart postgresql
-```
+## Create database tables
 
-* **Step 5** - Create the tables. For that we will use `create_db.py`.
+Please run these inside Python virtual environment
 
 ```sh
 python3 create_db.py
 # enter email and password
 python3 manage.py db stamp head
 ```
-**Note 1:** In case you made your own username and password in Step 2 are now getting `FATAL:  password authentication failed for user "john"` , probable cause is non updation of `.env` file. To resolve it, open the `.env` file and update `DATABASE_URL=postgresql://USERNAME:PASSWORD@127.0.0.1:5432/oevent` and you are good to go.
```

Comment:
This note is important. Since our .env file now has ident method of authentication, many people  on Mac will encounter this error because they will have created the database using username and password. So this should be kept